templates/webhook.bin.erb

```diff
@@ -160,11 +160,17 @@ end
     # The best we can do is just deploy all environments by passing nil to
     # deploy() if we don't know the correct branch.
     if prefix.nil? or prefix.empty? or branch.nil? or branch.empty?
-      deploy(normalize(branch))
+      env = normalize(branch)
     else
-      deploy(normalize("#{prefix}_#{branch}"))
+      env = normalize("#{prefix}_#{branch}")
     end
 
+    if ignore_env?(env)
+      $logger.info("Ignoring #{env} environment deploying")
```

Comment:
How about 'Skipping deployment of environment #{env} according to ignore_environments configuration' or similar?